README.md

```diff
@@ -13,6 +13,22 @@ Single-crate workspaces can use it as:
   run: # your `cargo publish` code here
 ```
 
+# Input options
+
+Every argument is optional.
+
+| Input              | Description                                                                                                                       | Default |
+|--------------------|-----------------------------------------------------------------------------------------------------------------------------------|---------|
+| crate-name         | The crate whose API to check for semver                                                                                           |         |
+| crate-target       | By default, check the library target of the crate. To check a different target (e.g. a binary target), set this to `--bin <NAME>` | `--lib` |
```

Comment:
Let's mark this flag as deprecated and try to remove it in the v2 of the action.

Bin targets don't have a library-like API so there isn't really anything for us to check. Once `cargo-semver-checks` merges into cargo, checking a bin target will be a hard error.